ui/tests/e2e/reporterSummary.md

```diff
@@ -11,41 +11,18 @@
 
 ⌛ _Duration: <%- minutes %> minutes and <%- seconds %> seconds_
 
-<% if (failed > 0) { %>
-  **Overall Result**: 👎 Some tests failed.
-<% } else { %>
-  **Overall Result**: 👍 All tests passed.
-<% } %>
+<%- failed > 0 ? "**Overall Result**: 👎 Some tests failed." : "**Overall Result**: 👍 All tests passed." %>
 
-<% if (testData && testData.length > 0) { %>
+<% if (testTable) { %>
 
 <details>
     <summary>[Show/Hide] Test Result Details</summary>
+    <div markdown="1">
```

Comment:
@sudhanshutech this is main addition to this. So that the Jekyll site can parse these block level elements correctly